### PR TITLE
Raspberry improvements

### PR DESCRIPTION
**Pull request by Waldz**

### CODE REVIEW

File: bin/package/installation/default [hunk 1]
```diff
@@ -2,6 +2,7 @@
 # Define additional args for mysterium_server (see help for full list)
 CONF_DIR="--config-dir=/etc/mysterium-node"
 RUN_DIR="--runtime-dir=/var/run/mysterium-node"
-DISCOVERY=""
+DAEMON_OPTS="--tequilapi.address=0.0.0.0"
+SERVICE_OPTS="openvpn"
```
Comment (soffokl) [inline bin/package/installation/default]: We are not going to support wireguard?

Comment (Waldz) [reply bin/package/installation/default]: I wanna make just 'openvpn' as default to Raspberry package

File: bin/package/installation/default [hunk 2]
```diff
@@ -2,6 +2,7 @@
 # Define additional args for mysterium_server (see help for full list)
 CONF_DIR="--config-dir=/etc/mysterium-node"
 RUN_DIR="--runtime-dir=/var/run/mysterium-node"
-DISCOVERY=""
+DAEMON_OPTS="--tequilapi.address=0.0.0.0"
```
[the remark this answers is not in the document]
Comment (soffokl) [reply bin/package/installation/default]: The motivation for this is still not clear for me.

[the remark this answers is not in the document]
Comment (Waldz) [reply bin/package/installation/default]: We want to control Raspberry node from same network

File: bin/package/installation/default [hunk 3]
```diff
@@ -1,7 +1,6 @@
...
-# Define additional args for mysterium_server (see help for full list)
+# Define additional args for `myst` service (see `myst --help` for full list)
 CONF_DIR="--config-dir=/etc/mysterium-node"
 RUN_DIR="--runtime-dir=/var/run/mysterium-node"
-DISCOVERY=""
-BROKER=""
-PROTO=""
+DATA_DIR="--data-dir=/var/lib/mysterium-node"
+DAEMON_OPTS=""
+SERVICE_OPTS=""
```
Comment (tadaskay) [inline bin/package/installation/default]: There should be LF at the end of the file

**Pull request merged**